gme/CMakeLists.txt

```diff
@@ -411,9 +411,9 @@ configure_file(gme_types.h.in gen_types.h @ONLY)
 set(TMP_PC_LIBS "")
 foreach(PC_LIB ${PC_LIBS})
     if(NOT "${TMP_PC_LIBS}" STREQUAL "")
-	set(TMP_PC_LIBS "${TMP_PC_LIBS} ${PC_LIB}")
+	   set(TMP_PC_LIBS "${TMP_PC_LIBS} ${PC_LIB}")
     else()
-	set(TMP_PC_LIBS "${PC_LIB}")
+	   set(TMP_PC_LIBS "${PC_LIB}")
```

Comment:
Whoops, two these lines actually go farther than multiple 4 spaces indenting :eyes:

Reply:
No idea how that happened, thanks for catching that